### PR TITLE
Added changes to enable the skipped test from API

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Enabled the full coverage of API test cases...

### CODE REVIEW

File: packages/api/test/e2e/tx.e2e.ts [hunk 1]
```diff
@@ -119,16 +120,25 @@ describe('e2e transactions', () => {
...
+            assetOptions
+          ))
+        .signAndSend(sudoPair, async ({events, status}: SubmittableResult) => {
           if (status.isFinalized) {
             const assetIdAfter = (await api.query.genericAsset.nextAssetId()) as AssetId;
             // expect
             expect(assetIdAfter.gt(assetIdBefore)).toBeTruthy();
-            expect(Number(assetIdAfter.toString())).toBeGreaterThan(reservedIdStart);
-            expect(Number(assetIdBefore.toString())).toBeGreaterThan(reservedIdStart);
+            // We don't know the reservedIdStart on node
```
Comment (holygits) [inline packages/api/test/e2e/tx.e2e.ts:139]: We can remove this test, it's kind of business logic vs. API functionality

Comment (KarishmaBothara) [reply packages/api/test/e2e/tx.e2e.ts:139]: sure

**Comment by KarishmaBothara** [comment]
Just want to check something.. was is it that the genesisHash will I run on my local using same image is 0x1eafcb166faa644bc1a874cb5a4575bf0d0c6f76019009f187c7585711844eb7 and with docker, it is 0x1eafcb166faa644bc1a874cb5a4575bf0d0c6f76019009f187c7585711844eb7?
Whenever free, please help..

**Comment by benxgao** [comment]
Could change cennznet image to `cennznet/cennznet@sha256:c8f5f7dd93b044db3c5ffff71344c292669cd788bc7ab4bdfa1d1c48090f312b`, and run `./scripts/circleci/run_automation_test.sh -t integration` to test it locally

**Comment by KarishmaBothara** [comment]
PR is good for review.. updated the genesis in static metadata

**Pull request merged**